docs/prow/production-cluster-configuration.md

```diff
@@ -0,0 +1,59 @@
+# Production Cluster Configuration
+
+## Overview
+
+This instruction provides the steps required to deploy a production cluster for Prow.
+
+## Prerequisites
+
+Use the following tools and configuration:
+
+- Kubernetes 1.10+ on GKE
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) to communicate with Kubernetes.
+- [gcloud](https://cloud.google.com/sdk/gcloud/) to communicate with Google Cloud Platform.
+- The `kyma-bot` GitHub account
+- [Kubernetes cluster](./prow-installation-on-forks.md/#provision-a-cluster)
+- Two Secrets in the Kubernetes cluster:
+    - `hmac-token` which is a Prow HMAC token used for validating GitHub webhooks.
+    - `oauth-token` which is a GitHub token with read and write access to the `kyma-bot` account.
+- Google Cloud Platform configuration that includes:
+    - A [global static IP address](https://cloud.google.com/compute/docs/ip-addresses/reserve-static-external-ip-address) with the `prow-production` name.
+    - A [DNS registry](https://cloud.google.com/dns/docs/quickstart#create_a_managed_public_zone) for the `status.build.kyma-project.io` domain that points to the `prow-production` address.
```

Comment:
It also needs two buckets. One with secrets and one for logs. See instruction for forks.